docs/en/samples/fuse_affinity.md

```diff
@@ -0,0 +1,222 @@
+# Demo - How to use Fuse NodeSelector 
+
+In Fluid, the remote files defined in the `Dataset` resource object are schedulable, which means that you can manage where remote files are cached on the Kubernetes cluster just like your pods. Pods that perform computations can access data files through the Fuse client. Fuse client can be deployed globally in Kubernetes cluster, we can limit the deployment scope of Fuse client through `nodeSelector`.
+
+This document will briefly show you the above features. Before that, you can visit [toc.md](https://github.com/fluid-cloudnative/fluid/blob/master/docs/en/TOC.md) to view the Fluid documentation to learn about the relevant knowledge details.
+
+## Installation
+
+Before running this example, please refer to the [Installation Documentation](https://github.com/fluid-cloudnative/fluid/blob/master/docs/en/userguide/install.md) to complete the installation, and check that each Fluid component is normal run:
+
+```
+$ kubectl get pod -n fluid-system
+alluxioruntime-controller-f87f54fd6-pqj77   1/1     Running   0          14h
+csi-nodeplugin-fluid-4pfmk                  2/2     Running   0          13h
+csi-nodeplugin-fluid-qwlm4                  2/2     Running   0          13h
+dataset-controller-bb95bb754-dsxww          1/1     Running   0          14h
+fluid-webhook-66b77ccb8f-vvqjb              1/1     Running   0          13h
+```
+
+Typically, you will see a pod named `dataset-controller`, a pod named `alluxioruntime-controller`, and several pods named `csi-nodeplugin` running. Among them, the number of `csi-nodeplugin` these pods depends on the number of nodes in your Kubernetes cluster.
+
+## Create new work environment
+
+```
+$ mkdir <any-path>/fuse-nodeselector-use
+$ cd <any-path>/fuse-nodeselector-use
+```
+
+## Demo
+
+**Check all nodes**
+
+```
+$ kubectl get nodes
+NAME                      STATUS   ROLES    AGE   VERSION
+cn-beijing.172.16.0.101   Ready    <none>   13h   v1.20.11-aliyun.1
+cn-beijing.172.16.0.99    Ready    <none>   23h   v1.20.11-aliyun.1
+```
+
+**Use labels to identify nodes**
+
+```
+$ kubectl label nodes cn-beijing.172.16.0.101 select-node=true
+node/cn-beijing.172.16.0.101 labeled
+```
+
+In the next steps, we will use the `NodeSelector` to manage the deployment scope of the Fuse client, so mark the desired node here.
+
+**Check nodes again**
+
+```
+$ kubectl get node -L select-node
+NAME                      STATUS   ROLES    AGE   VERSION             SELECT-NODE
+cn-beijing.172.16.0.101   Ready    <none>   13h   v1.20.11-aliyun.1   true
+cn-beijing.172.16.0.99    Ready    <none>   23h   v1.20.11-aliyun.1 
+```
+
+Currently, out of all 2 nodes, only one node has the `select-node=true` tag added, and next, we hope that the Fuse client will only be deployed on this node.
```

Comment:
IMO,  "label" is better than "tag" since it's the predefined concept in Kubernetes